src/pages/dashboard/RegistrationListItem.tsx

```diff
@@ -31,48 +35,52 @@ const RegistrationListItem: FC<RegistrationListItemProps> = ({ registration }) =
   const { t } = useTranslation('publicationTypes');
   const registrationId = registration.id.split('/').pop();
 
+  const focusedContributors = [...registration.contributors].splice(0, 5);
+  const countRestContributors = registration.contributors.length - focusedContributors.length;
+
   return (
     <ListItem divider>
-      <ListItemText
-        disableTypography
-        data-testid="result-list-item"
-        primary={
-          <Typography variant="h4">
-            <MuiLink component={Link} to={`/registration/${registrationId}/public`}>
-              {registration.title}
-            </MuiLink>
-          </Typography>
-        }
-        secondary={
-          <>
-            {registration.contributors &&
-              registration.contributors.map((contributor, index) => (
-                <Fragment key={index}>
-                  {contributor.id ? (
-                    <MuiLink component={Link} to={`/user/${contributor.id}`}>
-                      <StyledContributor>{contributor.name}</StyledContributor>
-                    </MuiLink>
-                  ) : (
-                    <StyledContributor>{contributor.name}</StyledContributor>
-                  )}
-                </Fragment>
-              ))}
-            <Typography>{registration.abstract}</Typography>
-            <StyledMetadata>
-              {registration.publishedDate && (
-                <>
-                  <CalendarIcon />
-                  <span>{displayDate(registration.publishedDate)}</span>
-                </>
+      <ListItemText disableTypography data-testid="result-list-item">
+        <Typography variant="h4">
+          <MuiLink component={Link} to={`/registration/${registrationId}/public`}>
+            {registration.title}
+          </MuiLink>
+        </Typography>
+        <StyledContributors>
+          {focusedContributors.map((contributor, index) => (
+            <Fragment key={index}>
+              {contributor.id ? (
+                <MuiLink component={Link} to={`/user/${contributor.id}`}>
+                  <Typography>{contributor.name}</Typography>
+                </MuiLink>
+              ) : (
+                <Typography>{contributor.name}</Typography>
               )}
-              <>
-                <TagIcon />
-                <span>{t(registration.publicationType)}</span>
-              </>
-            </StyledMetadata>
+            </Fragment>
+          ))}
+          {countRestContributors > 0 && (
+            <Typography>({t('common:x_others', { count: countRestContributors })})</Typography>
+          )}
+        </StyledContributors>
+
+        <Typography>
+          <Truncate lines={3} ellipsis="[...]">
+            {registration.abstract}
+          </Truncate>
+        </Typography>
+        <StyledMetadata>
+          {registration.publishedDate && (
```

Comment:
denne kan fjernes når https://unit.atlassian.net/browse/NP-1819 er ferdig

Reply:
...men kanskje burde vi bare beholde den uansett og legge til samme sjekk for `publicationType` også, for å være idiotsikker 🤔

Reply:
Har ikke full oversikt, men flere/bedre sjekker er vel bare positivt?